project/AppDependencies.scala

```diff
@@ -6,23 +6,22 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-27" % "4.2.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.57.0-play-27",
-    "uk.gov.hmrc"             %% "play-frontend-govuk"        % "0.69.0-play-27",
-    "uk.gov.hmrc"             %% "play-language"              % "4.12.0-play-27"
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.64.0-play-28",
+    "uk.gov.hmrc"             %% "play-frontend-govuk"        % "0.71.0-play-28",
+    "uk.gov.hmrc"             %% "play-language"              % "5.0.0-play-28"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"         % "4.2.0"            % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.3.0"            % "test, it",
     "org.scalatest"           %% "scalatest"                      % "3.0.8"            % "test, it",
```

Comment:
Version can be increased
```suggestion
    "org.scalatest"           %% "scalatest"                      % "3.0.9"            % "test, it",
```

Reply:
Sorted.